### DISCUSSION TITLE
which performance is better,  msquic on xdp or on dpdk？

### DISCUSSION THREAD

**Discussion by dreamwind1985** [microsoft/msquic#3965] [Q&A] [answered]
I saw the performance of msquic on xdp is much better than the performance of msquic using network protocol stack of linux kernel. 
Does anybody know the performance of msquic on dpdk, and comparing with the performance on xdp, which is better?

**Comment by nibanks** [comment, marked as answer]
We don't have any up to date data for MsQuic w/ DPDK, on Windows or Linux.
